config/initializers/sidekiq.rb

```diff
@@ -0,0 +1,9 @@
+Sidekiq.configure_server do |config|
+  # Calls to Rails.logger in a sidekiq process will use Sidekiq's logger
+  Rails.logger = Sidekiq::Logging.logger
+
+  config.on(:startup) do
+    already_scheduled = Sidekiq::JobSet.new.any? { |job| job.klass == "PostcodesCollectionWorker" }
```

Comment:
Checking for an already scheduled worker feels like a good idea, just wondering if you came across this in testing or you're just being cautious? I'd usually advise adding in a guard like this only if it becomes necessary, i.e. just have `PostcodesCollectionWorker.perform_async` in this block.